http-api/src/lib.rs

```diff
@@ -31,12 +39,18 @@ use librad::{git::types::Namespace, git::Urn, paths::Paths, profile::Profile, Pe
 use radicle_source::surf::file_system::Path;
 use radicle_source::surf::vcs::git;
 
+use crate::auth::{AuthMessage, AuthRequest, AuthState, UnauthorizedMessage};
 use crate::project::Info;
 
 use commit::{Commit, CommitContext, CommitTeaser, CommitsQueryString, Committer, Peer};
 use error::Error;
 
 pub const VERSION: &str = env!("CARGO_PKG_VERSION");
+pub const POPULATE_FINGERPRINTS_INTERVAL: u64 = 20;
+pub const CLEANUP_SESSIONS_INTERVAL: u64 = 60;
+pub const UNAUTHORIZED_SESSIONS_EXPIRATION: u64 = 60;
+// Authorized Sessions expire after 1 week
+pub const AUTHORIZED_SESSIONS_EXPIRATION: u64 = 604800;
```

Comment:
Expirations should be of type `std::time::Duration`, you can use `Duration::from_secs` and `Duration::as_secs` to convert.